package.json

```diff
@@ -72,19 +71,13 @@
     "react": "16.8.3",
     "react-dom": "16.8.3",
     "react-native": "https://github.com/expo/react-native/archive/sdk-35.0.0.tar.gz",
-    "react-native-web": "^0.11.1",
+    "react-native-web": "^0.13.18",
     "react-test-renderer": "16.8.3",
     "tslint": "5.12.0",
     "tslint-config-prettier": "1.17.0",
     "typescript": "3.4.5",
     "world-countries": "2.1.0"
   },
-  "peerDependencies": {
-    "react": "*",
-    "react-dom": "*",
-    "react-native": "*",
-    "react-native-web": "*"
-  },
```

Comment:
I'm here because I'm most interested in this change, I get peer-dependency warnings from this package, and npm v7+ will make maintaining these peer dependencies very important, and also difficult. Best practice to remove them these days.

Reply:
I am new to publishing so thought the versions need to change. Removed that from the commit. This is the part I tested works. Hence, this should be merged.